### PR TITLE
Fe profile details page

### PR DESCRIPTION
**Pull request by wiroger9595**
### What this PR does (required):
- create front end profile detail page make a page responsive
- user can send a request form to book 

### Screenshots / Videos (front-end only):
- desktop 
Screen Shot 2022-02-07 at 11 57 10 PM

- mobile
Screen Shot 2022-02-07 at 11 57 36 PM

### Any information needed to test this feature (required):
- request form time booking have yup that end time can't be shorter than start time and start time will base on local time on the computer
- yup will validate any base on date and 

### Any issues with the current functionality (optional):
-The navbar page has not responsive on mobile and navbar components. 
- have not add if pet sitter's time been book
- there have lots file changed, my change is in src/pages/ProfileDetails

### CODE REVIEW

**Review by ugur-akin — COMMENTED** [review]
Hello fellow contributor! I'm a robot and I'll be reviewing your PR for its Communication aspects *\*beep boop\**!

### Let's start with the pull request title:

Your title looks good overall, thanks for nicely formatting it! &#9989;


### Then there is the pull request summary:

Your summary looks good overall, thanks for paying attention to your communication! &#9989;



All in all, well-communicating pull requests is an important skill, hence we encourage all candidates to build strong habits in this regard. For additional information, [this article](https://hugooodias.medium.com/the-anatomy-of-a-perfect-pull-request-567382bb6067) touches on some more important points.
  
Good luck &#127881;

**Comment by ugur-akin** [comment]
Hi @wiroger9595 , sorry about the comment above 😅 It's part of a new feature we're working and I just needed a PR to test it on 👍🏼 

Cheers,

File: client/src/components/AvatarDisplay/AvatarDisplay.tsx [hunk 1]
```diff
@@ -4,10 +4,21 @@ import { User } from '../../interface/User';
...
-const AvatarDisplay = ({ user }: Props): JSX.Element => {
-  return <Avatar alt="Profile Image" src={`https://robohash.org/${user.email}.png`} />;
+const AvatarDisplay = ({ user, width, height, photoUrl }: Props): JSX.Element => {
+  if (photoUrl !== '') {
+    return <Avatar sx={{ width, height }} alt="Profile Image" src={photoUrl} style={{ top: '180px' }} />;
+  } else {
+    return (
+      <Avatar sx={{ width, height }} alt="Profile Image " style={{ top: '180px' }}>
+        {user.name[0]}
+      </Avatar>
```
Comment (jtingling) [inline client/src/components/AvatarDisplay/AvatarDisplay.tsx:19]: From a design perspective, would a generic avatar image be better like the pet sitter paw print?

File: client/src/pages/ProfileDetails/ProfileDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+
+  const updateWindowWidth = () => {
+    setWindowWidth(window.innerWidth);
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', updateWindowWidth);
+    return () => window.removeEventListener('resize', updateWindowWidth);
+  });
```
Comment (jtingling) [inline client/src/pages/ProfileDetails/ProfileDetails.tsx:32]: It looks like this code is used for resizing page components based on hardcoded breakpoints. Consider using standardized breakpoints provided with mui.

File: client/src/pages/ProfileDetails/ProfileDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <PageContainer>
+      <Grid
+        maxWidth={windowWidth < 800 ? 400 : windowWidth < 1200 ? 600 : 1200}
+        m={'auto'}
+        justifyContent="space-around"
+        container
+        alignItems="flex-start"
+      >
+        <Grid
+          mb={windowWidth < 1200 ? 3 : 0}
```
Comment (jtingling) [inline client/src/pages/ProfileDetails/ProfileDetails.tsx:44]: See comment above.

File: client/src/pages/ProfileDetails/useStyles.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      left: 'calc(50% - 55px)',
+      border: '5px solid ghostwhite',
+    },
+    '@media(min-width:600px)': {
+      minHeight: 300,
+      '& > *': {
+        top: 220,
+        left: 'calc(50% - 80px)',
+      },
+    },
```
Comment (jtingling) [inline client/src/pages/ProfileDetails/useStyles.tsx:22]: Mui provides `theme.breakpoints` for this.

**Comment by moffatethan** [comment]
@wiroger9595 Any ETA on these changes Rob?

**Review by moffatethan — CHANGES_REQUESTED** [review]
Why are we deleting the settings page?

**Review by moffatethan — CHANGES_REQUESTED** [review]
Lots of merge conflicts from the approved PRs, please be careful when addressing them!

**Pull request closed without merging**